src/features/projects/components/ProjectSnippet.tsx

```diff
@@ -33,6 +33,7 @@ export default function ProjectSnippet({
     : '';
 
   const { selectedPl, hoveredPl } = React.useContext(ProjectPropsContext);
+  const { tenantID } = React.useContext(ParamsContext);
```

Comment:
This can be grouped with line 29, as we're already using `ParamsContext` in this component.